### ISSUE TITLE
As A22 user I would like to see the web-cams on the highway through a specific web-component #3

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/webcomponents-brennerlec#5]
- [ ]  Design of the web-component

This could be simply done as in the A22 web-site:

![Image](https://github.com/user-attachments/assets/8d1c8bd5-11fd-4973-8921-e4efa5aec0d4)

- [ ] Implementation of the web-component -> @rcavaliere will provide here specific details for the implementation

The web-cams data can be retrieved at the following link (testing environment): https://api.tourism.testingmachine.eu/v1/WebcamInfo?source=a22

**Comment by rcavaliere** [comment]
@MaxXx1313 @sh-olmedo please start with the design of this new web-component. The implementation is already possible since the data is here is already open

**Comment by rcavaliere** [comment]
@MaxXx1313 @sh-olmedo I have updated in the user story description the end-point from which the web-cam data can be retrieved: https://api.tourism.testingmachine.eu/v1/WebcamInfo?source=a22 I have seen that actually we have also the GPS information, which means that we can completely reimplement the visualization available on the A22 web-site. Please let me know if you need something more for the design and implementation of this web-component

**Comment by MaxXx1313** [comment]
@rcavaliere Is there an endpoint to get route path points?

**Comment by rcavaliere** [comment]
@MaxXx1313 no, in this case we just have point coordinates, related to the positions of the web-cams

**Comment by MaxXx1313** [comment]
@rcavaliere Hi! We have some code to review related to this feature. I think it's not completely comply with the requirements yet, but it would be nice to verify if the approach is fine to you.

**Comment by MaxXx1313** [comment]
Here are screenshots of the result:
![Image](https://github.com/user-attachments/assets/354047b5-6e65-4955-a6af-4375b3559f55)

![Image](https://github.com/user-attachments/assets/b8c89d19-ffbf-43ba-88eb-f24643a2ae6b)

![Image](https://github.com/user-attachments/assets/87a340c1-6923-4075-aa32-94be0cd9dab2)

Also, with CSS it's possible to adjust most of the styles, for example to create dark mode:

![Image](https://github.com/user-attachments/assets/6c65aa6a-a0ed-4f43-87c7-285f0f5b4491)
